blocks/extensions/stopping.py

```diff
@@ -0,0 +1,35 @@
+from . import FinishAfter
+
+
+class FinishIfNoImprovementAfter(FinishAfter):
+    """Stop after improvements have ceased for a given period.
+
+    Parameters
+    ----------
+    iterations : int
```

Comment:
In fact, since monitoring is only done every epoch by default (#684), perhaps this extension (and `TrackTheBest`) ought to default to `after_epoch`, in which case waiting for either a number of iterations or a number of epochs makes perfect sense.